app/consumer-democracy/app.go

```diff
@@ -903,3 +904,16 @@ func initParamsKeeper(appCodec codec.BinaryCodec, legacyAmino *codec.LegacyAmino
 
 	return paramsKeeper
 }
+
+// MakeTestEncodingConfig creates an EncodingConfig for testing. This function
+// should be used only in tests or when creating a new app instance (NewApp*()).
+// App user shouldn't create new codecs - use the app.AppCodec instead.
+// [DEPRECATED]
```

Comment:
Why does this PR add functions that are marked as depreciated? Did this code already exist in some form?

Reply:
I copied it out of the last SPM removal I did with my own two hands.

Reply:
but -- to be clear, this is a deprecated way to do things and we have been trying to pull this out of the sdk for ages.

Reply:
this is a shim due to the design of ics